delivery_schenker/models/delivery_carrier.py

```diff
@@ -358,12 +358,18 @@ def _schenker_shipping_information_package_volume(self, picking, package):
             ]
         )
 
+    def _float_round(self, value, precision_digits):
+        return float_repr(
+            float_round(value, precision_digits=precision_digits),
```

Comment:
```suggestion
            float_round(value, precision_digits=precision_digits, rounding_method="UP"),
```
you need this in case the volume is <0.005, it will error due to rounding to 0.0 when the minimum is 0.01

Reply:
I don't think we should always round up, but better set a min on the volume

Reply:
does this min applies also to weight ?

Reply:
I'm not sure, looking at the schema I would say yes

The difference is that with a minimum of 0.01 then cases where volume is misconfigured to 0 would get through, it was something you mentioned a few months ago when the minimum was removed https://github.com/OCA/delivery-carrier/pull/675#discussion_r1260265410.

I think it's ok to error when the volume isn't configured properly, but the issue I encountered was that even when configured properly, sometimes the volume was just very small (<0.005) and it rounded to 0 and errored, hence I changed it to always rounding up with the idea that giving an overestimate would always be better for the carrier than an underestimate but the difference being at most 0.01 makes it feel pretty indifferent to the carrier

Reply:
I think rounding up is fine for volume and weight